.github/workflows/pr.yml

```diff
@@ -238,11 +238,33 @@ jobs:
           echo "Final collection_ids: $collection_ids"
           echo "collection_ids=${collection_ids}" >> $GITHUB_ENV
 
+      - name: Upload published collections files as directory artifact
+        uses: actions/upload-artifact@v4
+        env:
+          PUBLISHED_COLLECTION_FILES: ${{ needs.publish-new-datasets.outputs.publishedCollections }}
+          ARTIFACT_RETENTION_DAYS: ${{ vars.ARTIFACT_RETENTION_DAYS }}
```

Comment:
This is set to 10, but if not set, GitHub defaults it to 90 days.

Reply:
This is cool--I just opened the artifact from the last automated run!